ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-master.yaml

```diff
@@ -76,7 +76,7 @@ resources:
       cpu: 100m
       memory: 200Mi
 tag_specification:
-  name: "4.7"
+  name: "4.8"
```

Comment:
I _think_ all 4.8 imagestreams are not available yet.
I left this as 4.7 in cnf-features-deploy.